cargo-dist/templates/installer/homebrew.rb.j2

```diff
@@ -68,29 +68,29 @@ class {{ formula_class }} < Formula
 
   def install
     {%- if arm64_macos.binaries %}
-    on_macos do
-      on_arm do
+    if OS.mac?
```

Comment:
Like the audit you ran suggested, if we're making changes here anyway, we might as well combine these conditions onto a single line.

Reply:
makes sense. I combined the ifs but left it as a statement. It could be all one line like what the audit did, but I'm not sure how long the install lines are expected to get in practice.